src/routes/SafeAppLandingPage/SafeAppLandingPage.tsx

```diff
@@ -0,0 +1,321 @@
+                        {/* Create new Safe */}
+                        <BodyImage>
+                          <Img alt="Vault" height={92} src={SuccessSvg} />
+                        </BodyImage>
+
+                        <Button
+                          data-testid={'create-new-safe-link'}
+                          size="lg"
+                          color="primary"
+                          variant="contained"
+                          component={Link}
+                          to={OPEN_SAFE_ROUTE}
+                        >
+                          <Text size="xl" color="white">
+                            Create new Safe
+                          </Text>
+                        </Button>
+                      </>
+                    )}
+                  </>
+                ) : (
+                  <>
+                    {/* Connect your Wallet */}
+                    <StyledProvider>
+                      <Provider
+                        info={<ProviderDisconnected />}
+                        open={open}
+                        toggle={toggle}
+                        render={(providerRef) =>
+                          providerRef.current && (
+                            <Popper
+                              anchorEl={providerRef.current}
+                              style={{ zIndex: 1301 }}
+                              open={open}
+                              placement="bottom"
+                              popperOptions={{ positionFixed: true }}
+                            >
+                              <ClickAwayListener mouseEvent="onClick" onClickAway={clickAway} touchEvent={false}>
+                                <List component="div">
+                                  <ConnectDetails />
+                                </List>
+                              </ClickAwayListener>
+                            </Popper>
+                          )
+                        }
+                      />
+                    </StyledProvider>
+                  </>
+                )}
+              </UserSafeContainer>
+
+              {/* Demo Safe */}
+              <SafeDemoContainer>
+                <Title size={'xs'}>Want to try the app before using it?</Title>
+
+                <BodyImage>
+                  <Img alt="Demo" height={92} src={DemoSvg} />
+                </BodyImage>
+                {safeAppUrl && (
+                  <StyledDemoButton
+                    data-testid={'open-demo-app-link'}
+                    color="primary"
+                    component={Link}
+                    to={`${demoSafeAppsPath}?appUrl=${encodeURI(safeAppUrl)}`}
+                    size="lg"
+                    variant="outlined"
+                  >
+                    Try Demo
+                  </StyledDemoButton>
+                )}
+              </SafeDemoContainer>
+            </ActionsContainer>
+          </>
+        )}
+      </StyledCard>
+    </Container>
+  )
+}
```

Comment:
I know it's a draft, but some early feedback: this component is HUGE <img src="https://emoji.gg/assets/emoji/3284-thisistheworstmichael.png" height="30" />

Reply:
mmm all of these components are only used in this new Page, I can separate it in different components but I will suggest to keep it in this file. For me, separate components in different files only makes sense when you want to reuse them in other places.
What is your opinion about that?

Reply:
In the same file is totally fine. Although sometimes it leads to 20 imports mixed across them, and 20 styled components in the same scope.

Another approach is to create a folder, e.g. `src/components/SafeApps/LandingPage` and put those components into that folder in separate files.

Reply:
done!